packages/contracts/contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol

```diff
@@ -148,7 +159,7 @@ contract OVM_ECDSAContractAccount is iOVM_ECDSAContractAccount {
                     "Calls to self are disabled until upgradability is re-enabled."
                 );
 
```

Comment:
This will not work when the gasLimit is less than the intrinsic gas. Will need to set to 0 if `intrinsicGas > gasLimit`